tests/utils.go

```diff
@@ -3962,35 +3946,29 @@ func StartTCPServer(vmi *v1.VirtualMachineInstance, port int) {
 	Expect(err).ToNot(HaveOccurred())
 }
 
+func StartHTTPServerWithLoggedInVMI(vmi *v1.VirtualMachineInstance, port int, isFedoraVM bool) {
+	var httpServerMaker string
+
+	if isFedoraVM {
+		httpServerMaker = fmt.Sprintf("python3 -m http.server %d --bind ::0 &", port)
```

Comment:
Can we use the lowest common solution that fits all? Conditions are always bad, we should try to avoid them whenever possible.

Reply:
Done

Reply:
I meant to use `nc` for fedora as well.. not to have a dependency on the OS here.

Reply:
Let's try it.

Reply:
After rebase we don't have this issue anymore.